library/core/src/option.rs

```diff
@@ -70,10 +70,23 @@
 //! }
 //! ```
 //!
-//! This usage of [`Option`] to create safe nullable pointers is so
-//! common that Rust does special optimizations to make the
-//! representation of [`Option`]`<`[`Box<T>`]`>` a single pointer. Optional pointers
-//! in Rust are stored as efficiently as any other pointer type.
+//! # Representation
+//!
+//! Rust guarantees to optimize the following types `<T>` such that an
```

Comment:
```suggestion
//! Rust guarantees to optimize the following types `T` such that an
```
I never saw `<T>` as notation for a type, not sure what that is supposed to indicate.